src/ui/public/timepicker/kbn_global_timepicker.html

```diff
@@ -28,5 +31,6 @@
       <i aria-hidden="true" class="fa fa-clock-o"></i>
       <pretty-duration from="timefilter.time.from" to="timefilter.time.to"></pretty-duration>
     </button>
+    <i ng-click="foward()" class="time-nav-btn fa fa-arrow-right ng-scope"></i>
   </li>
 </ul>
```

Comment:
IMHO, the arrows work better on either side of the range, for the reasons mentioned by @jgowdyelastic